lib/IRGen/GenDecl.cpp

```diff
@@ -4163,7 +4163,8 @@ Address IRGenFunction::createAlloca(llvm::Type *type,
                                     const llvm::Twine &name) {
   llvm::AllocaInst *alloca =
       new llvm::AllocaInst(type, IGM.DataLayout.getAllocaAddrSpace(), ArraySize,
-                           alignment.getValue(), name, AllocaIP);
+                           llvm::MaybeAlign(alignment.getValue()), name,
+                           AllocaIP);
```

Comment:
Do we ever actually use `Alignment(0)` as input to this?

Reply:
I'm not certain, but I do agree with you that we shouldn't be passing 0 - the signature for `AllocaInst` has changed and now requires the `MaybeAlign` type (as a wrapper to the alignment).

Reply:
Right, I'm just wondering if maybe we can pass `llvm::Align(alignment.getValue())` (or even have an implicit conversion, and maybe someday unify the types).